openslides_backend/action/actions/motion/create.py

```diff
@@ -42,6 +42,10 @@ class MotionCreate(AmendmentParagraphHelper, MotionCreateBase):
         additional_optional_fields={
             "workflow_id": optional_id_schema,
             "submitter_ids": id_list_schema,
+            "amendment_paragraph": {
+                "type": "object",
+                "propertyNames": {"pattern": POSITIVE_NUMBER_REGEX},
```

Comment:
Also restrict the values here (currently, a value of e.g. `null` or `1` is also valid, when only strings shoul be valid). I would suggest to use `patternProperties` in combination with `additionalProperties: false`.

I would also extract this partial jsonschema definition and reuse it in `motion.update`.

Reply:
Done.